src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,12 @@
+import { TodoInfo } from '../TodoInfo';
+
 // Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+export const TodoList = ({ todos }) => (
+  <>
```

Comment:
There is no need to use fragments while you are rendering just one element inside